scripts/runhttpd

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/bash
 
+# shellcheck disable=SC1091
```

Comment:
Cannot we follow the recommendation in https://www.shellcheck.net/wiki/SC1090 instead of disabling it?

Reply:
`disable=SC1091` needs to be there, since shellcheck won't follow `source=xxx` as `/bin/xxx` is considered external source by shellcheck. We could use `SC1090` recommendation, if it was pointing to non-absolute path.